src/extras/amd.js

```diff
@@ -7,7 +9,10 @@
   const emptyInstantiation = [[], function () { return {} }];
 
   function unsupportedRequire () {
-    throw Error('AMD require not supported.');
+    if (DEV)
+      throw Error(errMsg(7, 'AMD require not supported.'));
+    else
+      throw Error(errMsg(7));
```

Comment:
These should be ok to use the logical form:

```suggestion
    throw Error(DEV ? errMsg(7, 'AMD require not supported.') : errMsg(7));
```